vllm/engine/arg_utils.py

```diff
@@ -48,6 +49,12 @@ def add_cli_args(
             type=str,
             default=EngineArgs.tokenizer,
             help='name or path of the huggingface tokenizer to use')
+        parser.add_argument(
+            '--revision',
+            type=str,
+            default=None,
+            help='The specific model version to use. It can be a branch '
+            'name, a tag name, or a commit id.')
```

Comment:
Nit: In the help message, can you add a description of the default behavior? Bascially add "By default, we use the main branch."